packages/core/upload/server/services/api-upload-folder.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const { isNil, get } = require('lodash/fp');
+const { getService } = require('../utils');
+
+const folderModel = 'plugin::upload.folder';
+const API_UPLOAD_FOLDER_BASE_NAME = 'Uploads';
```

Comment:
Just wondering if we should call it "API Uploads", since I'm also uploading something through the Admin UI, which might be confusing?

Reply:
@yanniskadiri what do you think?